receiver/k8sobjectsreceiver/receiver.go

```diff
@@ -122,8 +122,10 @@ func (kr *k8sobjectsreceiver) startPull(ctx context.Context, config *K8sObjectsC
 		select {
 		case <-ticker.C:
 			objects, err := resource.List(ctx, metav1.ListOptions{
-				FieldSelector: config.FieldSelector,
-				LabelSelector: config.LabelSelector,
+				FieldSelector:        config.FieldSelector,
+				LabelSelector:        config.LabelSelector,
+				ResourceVersion:      config.ResourceVersion,
+				ResourceVersionMatch: metav1.ResourceVersionMatchNotOlderThan,
```

Comment:
If the user doesn't set `ResourceVersion`, will it be the same behavior as before? If not, I'd expect setting `ResourceVersion` on `metav1.ListOptions` only if it's explicitly provided by the user.

Reply:
Done mate!
Indeed the default behavior would be different, so I'm checking if the value was explicitly configured by the user.

Reply:
Hi @dmitryax friendly reminder, don't know if you were table to check this.